common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -570,6 +569,20 @@
                                 gettext('Show posts by {username}.'), {username: username}
                             );
                             self.addSearchAlert(message, 'search-by-user');
+                            DiscussionUtil.safeAjax({
+                                url: DiscussionUtil.urlFor('followed_threads', response.users[0].id),
```

Comment:
First of all, I am not able to understand why we need to add followed thread here. Is this the reason users are not able to see their all posts? what about scenario where posts are missing other than search? Like reporter said in ticket that posts are missing while listing posts under any particular category. 
If we really need to add followed thread, happy to discuss it more on how we can best refactor this to add an additional ajax call.